src/routes/transactions/mappers/common/native-staking.mapper.ts

```diff
@@ -228,7 +206,10 @@ export class NativeStakingMapper {
           chainId: args.chainId,
           validatorsPublicKeys,
         });
-        value = stakes.reduce((acc, stake) => acc + Number(stake.rewards), 0);
+        value = stakes.reduce((sum, stake) => {
+          const reward = Number(stake.rewards);
+          return sum + (isNaN(reward) ? 0 : reward);
```

Comment:
You're right, It can't be `NaN` as it's being validated as `NumericString` on `StakeSchema`. I've reverted this change in [071ad93](https://github.com/safe-global/safe-client-gateway/pull/1895/commits/071ad93a4c9e2734305584309fc0608aa464f605), thanks!